cmake/Utils.cmake

```diff
@@ -166,7 +166,7 @@ function(caffe2_binary_target target_name_or_src)
   # libraries even if they are not part of the public dependency libs. As a
   # result, we will explicitly link the test against the Caffe2 dependency
   # libs. This should be changed in a future cleanup.
-  target_link_libraries(${__target} ${Caffe2_DEPENDENCY_LIBS})
+  # target_link_libraries(${__target} ${Caffe2_DEPENDENCY_LIBS})
```

Comment:
This is still needed at the moment for some tests on server. Could you uncomment it?

Reply:
ditto